MapLoaderFailoverTest.testLoadsAll_whenInitialLoaderNodeRemovedAfterLoading

```
java.lang.AssertionError: Cluster size is not correct expected:<2> but was:<3>
    at org.junit.Assert.fail(Assert.java:88)
    at org.junit.Assert.failNotEquals(Assert.java:834)
    at org.junit.Assert.assertEquals(Assert.java:645)
    at com.hazelcast.test.HazelcastTestSupport.assertClusterSize(HazelcastTestSupport.java:662)
    at com.hazelcast.map.mapstore.MapLoaderFailoverTest.testLoadsAll_whenInitialLoaderNodeRemovedAfterLoading(MapLoaderFailoverTest.java:112)
```

https://hazelcast-l337.ci.cloudbees.com/view/Official%20Builds/job/Hazelcast-3.x-IbmJDK1.6/com.hazelcast$hazelcast/655/testReport/junit/com.hazelcast.map.mapstore/MapLoaderFailoverTest/testLoadsAll_whenInitialLoaderNodeRemovedAfterLoading/

--- a/hazelcast/src/test/java/com/hazelcast/map/mapstore/MapLoaderFailoverTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/map/mapstore/MapLoaderFailoverTest.java
@@ -108,7 +108,7 @@ public class MapLoaderFailoverTest extends HazelcastTestSupport {
         assertSizeAndLoadCount(map);
 
         hz3.getLifecycleService().terminate();
-        assertClusterSize(2, nodes[0]);
+        assertClusterSizeEventually(2, nodes[0]);
         map.loadAll(true);
 
         assertSizeEventually(MAP_STORE_ENTRY_COUNT, map);
@@ -132,7 +132,7 @@ public class MapLoaderFailoverTest extends HazelcastTestSupport {
         pausingLoader.awaitPause();
 
         hz3.getLifecycleService().terminate();
-        assertClusterSize(2, nodes[0]);
+        assertClusterSizeEventually(2, nodes[0]);
 
         pausingLoader.resume();
 

--- a/hazelcast/src/test/java/com/hazelcast/test/HazelcastTestSupport.java
+++ b/hazelcast/src/test/java/com/hazelcast/test/HazelcastTestSupport.java
@@ -681,7 +681,7 @@ public abstract class HazelcastTestSupport {
             @Override
             public void run()
                     throws Exception {
-                assertEquals("the size of the cluster is not correct", expectedSize, instance.getCluster().getMembers().size());
+                assertClusterSize(expectedSize, instance);
             }
         }, timeoutSeconds);
     }

